main.go

```diff
@@ -30,26 +33,28 @@ const (
 func applyVirtualMachineExport(vmName string) error {
 	log.Println("Applying VirtualMachineExport object...")
 
-	yaml := fmt.Sprintf(`
-apiVersion: export.kubevirt.io/v1alpha1
-kind: VirtualMachineExport
-metadata:
-  name: %s
-spec:
-  source:
-    apiGroup: "kubevirt.io"
-    kind: VirtualMachine
-    name: %s
-`, vmName, vmName)
-	cmd := exec.Command("kubectl", "apply", "-f", "-")
-	cmd.Stdin = strings.NewReader(yaml)
-	cmd.Stdout = os.Stdout
-	cmd.Stderr = os.Stderr
-
-	if err := cmd.Run(); err != nil {
+	client, err := kubecli.GetKubevirtClient()
+	if err != nil {
 		return err
 	}
-	return nil
+
+	kvApiGroup := "kubevirt.io"
+	vmExport := &v1alpha1.VirtualMachineExport{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: vmName,
+		},
+		Spec: v1alpha1.VirtualMachineExportSpec{
+			Source: corev1.TypedLocalObjectReference{
+				APIGroup: &kvApiGroup,
+				Kind:     "VirtualMachine",
```

Comment:
do not define these strings (kind, groups, ...) in your code. Instead use constants available in kubevirt package

Reply:
Sure thing.

Reply:
I don't see where is the constant of `"VirtualMachine"`. From a brief look at `kubevirt/kubevirt` codebase, I see that most of the code just using the string itself like here, and there is no constant variable for it.

Reply:
you can use https://github.com/kubevirt/kubevirt/blob/main/staging/src/kubevirt.io/api/core/v1/register.go#L79
and then call it as kubevirtv1.VirtualMachineGroupVersionKind.Kind